colaraz/lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -467,9 +467,9 @@ <h4 class="message-title">${_('Your current verification will expire soon.')}</h
                 </b><br>
                   ${Text(_("It's official. It's easily shareable. "
                       "It's a proven motivator to complete the course. {line_break}"
-                      "{link_start}Learn more about the verified {cert_name_long}{link_end}.")).format(
+                      "{link_start}Learn more about the verified {cert_name_long}{link_end}")).format(
                         line_break=HTML('<br>'),
-                        link_start=HTML('<a href="{}" class="verified-info" data-course-key="{}">').format(
+                        link_start=HTML('<a hidden href="{}" class="verified-info" data-course-key="{}">').format(
```

Comment:
why not just remove it though?

Reply:
As the client said we don't need this link right now. But in the future, he may need it.